metrics/prometheus/client.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2017 Mark Nevill. All Rights Reserved.
+// See LICENSE for licensing terms.
+
+package http_prometheus
+
+import (
+	"net/http"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/mwitkow/go-httpwares/metrics"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	clientStarted = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "http_client_started_requests_total",
```

Comment:
We avoids namespace/subsystem on purpose? (I think is totally the same than your way of specifying it)